v2/bigtable-changestreams-to-hbase/src/main/java/com/google/cloud/teleport/v2/templates/BigtableChangeStreamsToHBase.java

```diff
@@ -102,9 +102,9 @@ public interface BigtableToHbasePipelineOptions
         helpText =
             "Whether bidirectional replication between hbase and bigtable is enabled, adds additional logic to filter out hbase-replicated mutations")
     @Default.Boolean(false)
-    boolean getBidirectionalReplicationEnabled();
+    Boolean getBidirectionalReplicationEnabled();
 
-    void setBidirectionalReplicationEnabled(boolean bidirectionalReplicationEnabled);
+    void setBidirectionalReplicationEnabled(Boolean bidirectionalReplicationEnabled);
 
     @TemplateParameter.Text(
```

Comment:
Can be changed to `@TemplateParameter.Boolean`?

Reply:
No, this is Text (e.g. "BIDIRECTIONAL_REPL_SOURCE_CBT")